app/controllers/admin/settings/mandatories_controller.rb

```diff
@@ -0,0 +1,39 @@
+module Admin
+  module Settings
+    class MandatoriesController < Admin::ApplicationController
+      def create
+        errors = upsert_config(settings_params)
+
+        if errors.none?
+          Audit::Logger.log(:internal, current_user, params.dup)
+          redirect_to admin_config_path, notice: "Site configuration was successfully updated."
+        else
+          redirect_to admin_config_path, alert: "😭 #{errors.to_sentence}"
+        end
+      end
+
+      def upsert_config(configs)
+        errors = []
+        configs.each do |key, value|
+          settings_model = ::Settings::Mandatory::MAPPINGS[key.to_sym]
+          if value.is_a?(Array)
+            settings_model.public_send("#{key}=", value.reject(&:blank?)) if value.present?
+          else
+            settings_model.public_send("#{key}=", value.strip) unless value.nil?
+          end
+        rescue ActiveRecord::RecordInvalid => e
+          errors << e.message
+          next
```

Comment:
Uhm, yes it will, this is mostly a leftover of the old code from `SiteConfig::Upsert` that was copied into several controllers. Right now there's a lot of unnecessary duplication between the various controllers which I'm going to tackle pretty soon. If it's all the same to you I'd rather address all of that together.